src/main/java/cz/muni/fi/pa165/bluebat/entity/Trip.java

```diff
@@ -28,31 +19,30 @@
 @Entity
 @Setter
 @ToString
+@Getter
 public class Trip {
 
     @Id
     @GeneratedValue(strategy= GenerationType.IDENTITY)
-    @Getter
     private Long id;
 
-    @Getter
+
     private String name;
 
     @Column(nullable = false)
-    @Getter
+    @Future
     private LocalDate dateFrom;
 
     @Column(nullable = false)
-    @Getter
+    @Future
     private LocalDate dateTo;
 
     @Column(nullable = false)
-    @Getter
     private String destination;
```

Comment:
Taky bych použil NotBlank, všiml jsem si totiž, že to testuješ. Procházejí ti potom testy?